### PR TITLE
Documenting styles configuration for IE11 and autoprefixer need

### LINKED ISSUE #943: Update grid and themes topics with autoprefixer compilation description
Update both main grid topic and themes topic with autoprefixer compilation description.

### PR DESCRIPTION
**Pull request by HristoP96**
Closes #943

### CODE REVIEW

File: en/components/grid.md [hunk 1]
```diff
@@ -64,6 +64,29 @@ The **id** property is a string value and is the unique identifier of the grid w
...
+
+ In **angular.json** go to the **styles** property and add **igniteui-angular.css** file, which contains all the necessary prefixed styles:
+
+ ```json
+ //angular.json
+ ...
+    "styles": [
+        "src/styles.scss",
+        //Set igniteui-angular.css as a global style for your app  
+        "node_modules/igniteui-angular/styles/igniteui-angular.css"
```
[the remark this answers is not in the document]
Comment (HristoP96) [reply en/components/grid.md]: @simeonoff Good point! I will split this  styling configuration section into 2 parts: for CSS users and for Sass users.

[the remark this answers is not in the document]
Comment (simeonoff) [reply en/components/grid.md]: I don't think you need a CSS section.

[the remark this answers is not in the document]
Comment (HristoP96) [reply en/components/grid.md]: Ok

[the remark this answers is not in the document]
Comment (HristoP96) [reply en/components/grid.md]: Changes Done.

**Pull request merged**